harness/determined/common/schemas/expconf/_gen.py

```diff
@@ -1143,11 +1143,6 @@
                 "string",
```

Comment:
You also need to allow a list of strings

Reply:
made the change. 👍

Reply:
done